README.md

```diff
@@ -18,12 +18,12 @@ The exact contents of both of the above options can be obtained directly by clic
 
 ## Usage example
 ```
-from simtrack import Simtrack
+from simvue import Simvue
 
 ...
 
 # Using a context manager means that the status will be set to completed automatically
-with Simtrack() as run:
+with Simview() as run:
```

Comment:
Should be Simvue?

Reply:
Yes. For some reason I kept on writing Simview not Simvue :-)

Reply:
Fixed now